install/tools/certs/Makefile

```diff
@@ -38,7 +40,27 @@ help: Makefile
 	@sed -n 's/^##//p' $<
 
 #------------------------------------------------------------------------
-##root-ca:	generate root CA files (key and certifcate) in current directory
+##fetch-root-ca:	fetch root CA  and key from a k8s cluster.
+.PHONY: fetch-root-ca
+rawcluster := $(shell kubectl config current-context)
+cluster := $(subst /,-,$(rawcluster))
+pwd := $(shell pwd)
+fetch-root-ca:
+	@echo "fetching root ca from k8s cluster: "$(cluster)""
+	@@mkdir -p $(pwd)/$(cluster)
+	@KUBECONFIG=$(KUBECONFIG) kubectl get secret istio-ca-secret  -n istio-system -o "jsonpath={.data['ca-cert\.pem']}" | base64 -d > $(cluster)/k8s-root-cert.pem
```

Comment:
I would add a target for fetching the root CA from the alternate (and recommended) "setup Istio with an existing key". 

IMO we should just have a single secret for both modes, there is no value in the extra flags and confusion.